### DISCUSSION TITLE
HW 4 Number 2 Part C

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#60] [Q&A]
Confused on why I got this error and how to fix it

![image](https://user-images.githubusercontent.com/106542680/175837611-e2137ee2-b5e9-4bf6-a476-d13578ac9dd0.png)

**Comment by cooperrc** [comment]
The `ValueError` is coming up because `y` is defined in a strange way. 

I _think_ you want to use `x = solveLU(L, U, np.array([0, F[i], 0]))`

I loop through the `i` values from 0 -> `len(F)`
